src/test/ui/rust-2018/issue-54400-unused-extern-crate-attr-span.stderr

```diff
@@ -1,11 +1,8 @@
 error: unused extern crate
   --> $DIR/issue-54400-unused-extern-crate-attr-span.rs:12:1
    |
-LL | / #[cfg(blandiloquence)]
-LL | | extern crate edition_lint_paths;
-   | | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^-
-   | |________________________________|
-   |                                  help: remove it
+LL | extern crate edition_lint_paths;
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: remove it
```

Comment:
This is a regression, this test won't (shouldn't?) pass with the span change.

Reply:
I think it actually *will* pass with the span change, but it is still a regression. The reason it will pass is because the `#[cfg]` will end up being applied to the `fn main() {}` that is directly below (from a whitespace- and comment-ignoring point of view) this `extern crate`.

Probably the test should be changed to use `#![crate_type = "lib"]` and to remove `fn main() {}` so it will fail if there's a future regression like this.